src/main/ts/utils/ObjUtils.ts

```diff
@@ -9,9 +9,18 @@ export const lookup = <T>(o: T, k: keyof T): Option<T[keyof T]> =>
   hasKey(o, k) ? O.some(o[k]) : O.none;
 
 export const map = <A, B>(o: Record<string, A>, f: (a: A) => B): Record<string, B> => {
-  const r: Record<string, B> = {};
+  const r = {} as Record<string, B>;
   for (const k of Object.keys(o)) {
     r[k] = f(o[k]);
   }
   return r;
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const fromPairs = <K extends keyof any, V> (pairs: Array<[K, V]>): Record<K, V> => {
+  const r = {} as Record<K, V>;
```

Comment:
Following on from above comment - I couldn't write this as `const r: Record<K, V> = {}`. Anyone know why?

Reply:
Not sure but `K extends keyof any` seems very odd to me. What's the goal with that?

Reply:
I'm trying to say "any Record". 

Record is defined like this:

```
type Record<K extends keyof any, T> = {
    [P in K]: T;
};
```

Reply:
Is there any case where `K` does not extend `string`?

Reply:
No

Reply:
Maybe it's simpler to just use string then.